src/FieldSet/styles.js

```diff
@@ -0,0 +1,49 @@
+import css from 'styled-jsx/css'
+
+import { spacers, theme } from '../theme.js'
+
+export default css`
+    fieldset {
+        border: none;
+        margin: 0;
+        padding: 0;
+    }
+    legend {
+        font-size: 16px;
+        color: ${theme.default};
+        margin: 0 0 ${spacers.dp12};
```

Comment:
I think using the four-value sequence is preferable from a expressive perspective: `0 0 ${spacers.dp12} 0`